cms/private_media/tests/test_storages.py

```diff
@@ -0,0 +1,60 @@
+from unittest import mock
+
+from botocore.exceptions import ClientError
+from django.conf import settings
+from django.test import TestCase, override_settings
+from moto import mock_aws
+from wagtail_factories import DocumentFactory
+
+from cms.private_media.storages import PrivacySettingS3Storage
+
+
+@mock_aws
+@override_settings(AWS_STORAGE_BUCKET_NAME="test-bucket")
+class PrivacySettingS3StorageTests(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.document = DocumentFactory()
+
+    def setUp(self):
+        super().setUp()
+        self.storage = PrivacySettingS3Storage()
+        # Create a bucket to allow '_set_file_acl' to at least reach the 'put' stage
+        bucket = self.storage.connection.Bucket(settings.AWS_STORAGE_BUCKET_NAME)
+        bucket.create()
+
+    def test_make_private_client_error_on_acl_set(self):
+        with self.assertLogs("cms.private_media.storages", level="WARNING") as logs:
+            self.assertFalse(self.storage.make_private(self.document.file))
+        self.assertIn(
+            f"WARNING:cms.private_media.storages:Failed to set ACL for {self.document.file.name}: ",
+            logs.output[0],
+        )
+
+    def test_make_private_client_error_on_acl_get(self):
+        object_mock = mock.MagicMock()
+        object_mock.Acl.side_effect = ClientError({"Error": {"Code": "AccessDenied"}}, "GetObjectAcl")
+        # initialize the bucket to allow it to be patched
+        self.storage.bucket  # noqa: B018 # pylint: disable=pointless-statement
+        with mock.patch.object(self.storage._bucket, "Object", return_value=object_mock):  # pylint: disable=protected-access  # noqa: SIM117
```

Comment:
Issue: The point of using `mock_aws` is that this isn't necessary

Reply:
I'm no expert on `mock_aws`, but I couldn't figure out how to easily make it throw errors at different stages of the process like I need it to.

Reply:
The main reassurance we are getting from `mock_aws` is actually to test that it can get as far as the ACL setting part (that we're using the right things from boto).

Reply:
Hmm, yeah I'm not sure how to mock getting errors back from the API, other than emulating a file not existing. Emulating IAM issues is probably far more complex.